anesthetic/plot.py

```diff
@@ -343,6 +337,11 @@ def contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
         x and y coordinates of uniformly weighted samples to generate kernel
         density estimator.
 
+    levels: list
+        amount of mass within each iso-probability contour.
+        If None defaults to hist2d.
+        optional, default [0.68, 0.95]
```

Comment:
> `If None defaults to hist2d`.

Is this a copy-paste error or is that really how it works for `contour_plot_2d`?

Reply:
copy-paste error -- corrected.